politeiawww/api/v1/v1.go

```diff
@@ -92,12 +96,15 @@ const (
 	PropStatusNotReviewed PropStatusT = 2 // Proposal has not been reviewed
 	PropStatusCensored    PropStatusT = 3 // Proposal has been censored
 	PropStatusPublic      PropStatusT = 4 // Proposal is publicly visible
-
-	// Error contexts
-	ErrorContextProposalInvalidTitle = ValidProposalNameRegExp
 )
 
 var (
+
+	// PolicyProposalNameSupportedCharacters is the regular expression of a valid
+	// proposal name
+	PolicyProposalNameSupportedCharacters = []string{
+		"A-z", "0-9", "&", ".", ",", ":", ";", "-", " ", "@", "+", "#"}
```

Comment:
We probably need a slash "/" as well.  Maybe some others like bang "!" and parenthesis "(", ")".